deps/media-playback/media-playback/media.c

```diff
@@ -474,6 +474,7 @@ void mp_media_next_video(mp_media_t *m, bool preload)
 	frame->height = f->height;
 	frame->max_luminance = d->max_luminance;
 	frame->flip = flip;
+	frame->flags = 0;
 	frame->flags |= m->is_linear_alpha ? OBS_SOURCE_FRAME_LINEAR_ALPHA : 0;
```

Comment:
I'm fine with this change, but I think I decided on this in case other flags are added. Changing `|=` to `=` would be the same here only because there is currently only one flag?

Reply:
Correct. And I doubt other flags will be added.

Reply:
Okay